packages/components/psammead-brand/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 1.1.0   | [PR#528](https://github.com/bbc/psammead/pull/528) Add link prop to allow dynamic link |
```

Comment:
This should be a major breaking change. You're adding a required prop `link`.

Reply:
Should it definitely be a required prop?

Reply:
You are right. Since `Brand` doesn't have a default `url` it links to.